play/settings/base.py

```diff
@@ -25,34 +23,24 @@ def get_env(key, default=None, allow_default=True):
         raise NotImplementedError("Environment variable is unset: '%s'" % key)
     return os.environ[key]
 
-
-
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 # Get environment variable ENV from the system
 # default to production if it doesn't exist
 ENV = get_env('ENV', 'production', True)
 
-def is_local_env():
```

Comment:
This function was not being used.

We _may_ want to consider using [DJANGO_SETTINGS_MODULE](https://docs.djangoproject.com/en/2.1/topics/settings/#designating-the-settings) in the future. I think its a nice way to provide variations for different environments.